src/OSVRTrackedDevice.cpp

```diff
@@ -436,6 +436,16 @@ vr::DistortionCoordinates_t OSVRTrackedDevice::ComputeDistortion(vr::EVREye eye,
     coords.rfBlue[0] = coords_blue[0];
     coords.rfBlue[1] = 1.0f - coords_blue[1];
 
+	// offset for center of projection
+	auto x_offset = 0.5 - distortion_parameters.m_distortionCOP[0];
+	auto y_offset = 0.5 - distortion_parameters.m_distortionCOP[1];
+	coords.rfRed[0] = coords.rfRed[0] + x_offset;
```

Comment:
You're right, and it's not converted to overfill space either.

I could convert to overfill space and clamp to [0,1], but I think I'm barking up the wrong tree. It still doesn't explain why the COP offset in https://github.com/sensics/OSVR-RenderManager/blob/master/osvr/RenderKit/DistortionCorrectTextureCoordinate.h#L244 doesn't solve it, or why it displays differently in OSVR native RenderManager programs and SteamVR.